src/routes/images.rs

```diff
@@ -78,10 +78,11 @@ async fn thumbnail(
   let (size, file) = parts.into_inner();
 
   let url = format!(
-    "{}/image/thumbnail{}/{}",
+    "{}/image/process.{}?src={}&thumbnail={}",
     Settings::get().pictrs_url,
+    "jpg", // this can be changed to png or webp
```

Comment:
Might be good to make it identical to the original image?

Reply:
This can just be a forwarded param from the request. Now that pict-rs supports multiple formats, I can add webp with a jpg fallback like:

```
<picture>
  <source srcset="img/awesomeWebPImage.webp" type="image/webp">
  <source srcset="img/creakyOldJPEG.jpg" type="image/jpeg"> 
  <img src="img/creakyOldJPEG.jpg" alt="Alt Text!">
</picture>
```

Reply:
I think webp with a fallback to jpg is a good strategy. Unless you really care about thumbnails keeping transparency you shouldn't generate png thumbnails imo